### PR TITLE
Fix CLI build

### PR DESCRIPTION
**Pull request by JakeGinnivan**

### CODE REVIEW

File: apps/cli/package.json [hunk 1]
```diff
@@ -2,11 +2,10 @@
     "name": "@featureboard/cli",
     "version": "0.2.1",
     "license": "MIT",
-    "dependencies": {
+    "devDependencies": {
```
Comment (emlyn-rice) [inline apps/cli/package.json:5]: These we're added as they were need to run the package when it was run via npx if so they are dependencies or if they are not needed any more then we can just remove them.

File: apps/cli/project.json [hunk 2]
```diff
@@ -24,16 +59,6 @@
                 }
             }
         },
-        "package": {
```
Comment (emlyn-rice) [inline apps/cli/project.json:27]: If we don't package then only the ts will end up in the npm package as build isn't part of our pipeline

File: apps/cli/package.json (hunk 1 above)
Comment (JakeGinnivan) [reply apps/cli/package.json:5]: They are needed to be installed when we bundle, can move them to the root of the repo but I think it makes sense to keep them local to this package.

We now bundle these deps into a single .js file, so they are not needed to run the CLI, but still needed at build time

File: apps/cli/project.json (hunk 2 above)
Comment (JakeGinnivan) [reply apps/cli/project.json:27]: Nice catch

**Review by emlyn-rice — APPROVED** [review]

**Pull request merged**